config/config.go

```diff
@@ -644,6 +649,7 @@ var defaultConf = Config{
 	EnableBatchDML:               false,
 	CheckMb4ValueInUTF8:          true,
 	MaxIndexLength:               3072,
+	IndexLimit:                   64,
```

Comment:
better use `DefIndexLimit` here

Reply:
We use `DefIndexLimit` as a constant. I think it is ok to use `IndexLimit` here, as other configuration items do not start with `Def` @AilinKid